client/dashboard/app/queries.ts

```diff
@@ -98,3 +101,47 @@ export function profileMutation() {
 		},
 	};
 }
+
+export function siteSettingsQuery( siteId: string ) {
+	return {
+		queryKey: [ 'site-settings', siteId ],
+		queryFn: () => {
+			return fetchSiteSettings( siteId );
+		},
+		refetchOnWindowFocus: false,
+		retry: false,
+		enabled: !! siteId,
+	};
+}
+
+export function basicMetricsQuery(
+	url: string,
+	isLoadingSiteSettings: boolean,
+	cachedHash: string
+) {
+	return {
+		queryKey: [ 'url', 'basic-metrics', url ],
+		queryFn: () => {
+			return fetchBasicMetrics( url );
+		},
+		refetchOnWindowFocus: false,
```

Comment:
This seems implementation specific, it doesn't belong here?

Reply:
This was part of the [original prototype](https://github.com/Automattic/wp-calypso/pull/102573) and not added in this PR. It feels logical for this to be here, but I don't have a strong preference. 

@youknowriad thoughts?

Reply:
For me, it's for specific queries to override the global defaults but we should be doing it deliberately. I believe @ntsekouras might have added this originally. But I'm tempted to just use the defaults.

Reply:
> I know it pre-dates the PR. What I mean is that I don't think it belong here, I think we should add these options wherever the query is used?

Understood. So if I use the query in `Site Overview > Performance Cards` and `Site Performance > Report` for example (which will be the case), would you expect to pass query-specific params?

Reply:
Or `useQuery( { ...getQuery(), refetchOnWindowFocus, enabled } )`.

Reply:
That works for me.

Reply:
I had used `refetchOnWindowFocus` there because the request to that would trigger the creation of new reports, which is not something we want for this endpoint.